static/elements/chromedash-process-overview.js

```diff
@@ -142,9 +158,29 @@ class ChromedashProcessOverview extends LitElement {
     const label = action.name;
     const url = (action.url.replace('{feature_id}', this.feature.id)
       .replace('{outgoing_stage}', stage.outgoing_stage));
+    const checkCompletion = () => {
+      let foundEmailItem = false;
+      const pendingItems = stage.progress_items.filter(
```

Comment:
Rather than checking all the progress items in a stage, we should check all the progress item names in `action.prerequisites`.   That's because not all progress items are needed for a given action, and at the same time, actions can require progress items from earlier stages.  By looping over `action.prerequisites`, you would no longer need foundEmailItem because the prerequisites are named explicitly rather than depending on a naming convention.